internal/testbed/integration/filtering_scenario.go

```diff
@@ -0,0 +1,76 @@
+package integration
+
+import (
+	"testing"
+	"time"
+
+	"github.com/Dynatrace/dynatrace-otel-collector/internal/testcommon/testutil"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/testbed"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	defaultProcessors = map[string]string{
+		"memory_limiter": `
+  memory_limiter:
+    check_interval: 1s
+    limit_percentage: 100
+`,
+		"batch": `
+  batch:
```

Comment:
do we need the batch processor here? As we are sending only a small number of data items I think we might slow down the tests as the queue will not be full until the 10 second timeout has been reached

Reply:
the scenarios added to the test are just to showcase how the tests should be build, we can definitely remove the scenarios that are currently present, since they are not filtering anything. I just wanted to proof that the setup works and can be therefore used for filtering testing